src/custom/pages/Faq/index.tsx

```diff
@@ -415,6 +466,21 @@ export default function Faq() {
             allow you to wrap and unwrap ETH into WETH without needing to leave the dapp’s UI
           </p>
 
+          <h3 id="why-is-it-more-troublesome-to-trade-eth-to-erc20-than-erc20-to-eth">
+            Why is it more troublesome to trade ETH to ERC20 token, than ERC20 token to ETH?
```

Comment:
I feel this question is too long. Maybe:

`Why selling ETH is more troublesome?`

Reply:
I would go further and say: "Why can't I sell ETH?"

Reply:
agree with @alfetopito , less negative and just matter of fact

Reply:
I'm fine either way. Happy as long as it's short

Reply:
ok agree, @avsavsavs go with your gut